api/app/references/models.py

```diff
@@ -34,9 +35,29 @@ def __init__(self,
         self.lastname = lastname
         self.relation = relation
         self.email = email
+        self.reference_submitted = False
```

Comment:
I understand why you've added this, but I'm not the biggest fan of having a flag like this as it can easily become out of sync with the "truth" which is that a reference exists if and only if a record exists in the Reference table with a corresponding `reference_request_id`, so would prefer if the logic checked explicitly for that, even though it's another DB query.  What do you think  @amritpurshotam  ?

Reply:
I agree. @bamtak I think setup a relationship between `Reference` and `ReferenceRequest` and then you will be able to check if a reference exists. So on `ReferenceRequest` add the following line 
`references = db.relationship('Reference')` 
and on `Reference` add 
`reference_request = db.relationship('ReferenceRequest', foreign_keys=[reference_request_id])`

Reply:
Also make sure to run another migration to make sure this column gets dropped